src/background/details-view-controller.ts

```diff
@@ -1,9 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { DetailsViewControllerType } from 'background/details-view-controller-type';
 import { BrowserAdapter } from '../common/browser-adapters/browser-adapter';
 import { DictionaryStringTo } from '../types/common-types';
 
-export class DetailsViewController {
+export class DetailsViewController implements DetailsViewControllerType {
```

Comment:
I felt a little confused by this naming. Maybe better would be `ExtensionDetailsViewController implements DetailsViewController`? Or is `DetailsViewController` used in too many places to want to do the refactoring in the same PR?

Reply:
updated